LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015
+Copyright (c) 2016
```

Comment:
As I learned from @michaeldupuisjr , this should still be 2015 to make sure everything for that year is also copyrighted